core/store/migrate/migrations/0232_log_poller_word_topic_indexes.sql

```diff
@@ -0,0 +1,62 @@
+-- +goose Up
+
+drop index if exists evm.evm_logs_idx_data_word_one;
+drop index if exists evm.evm_logs_idx_data_word_two;
+drop index if exists evm.evm_logs_idx_data_word_three;
+drop index if exists evm.evm_logs_idx_data_word_four;
+drop index if exists evm.evm_logs_idx_topic_two;
+drop index if exists evm.evm_logs_idx_topic_three;
+drop index if exists evm.evm_logs_idx_topic_four;
+
+create index evm_logs_idx_data_word_one
```

Comment:
I missed this before, but I think you need `evm.` in front of all of these. I'm guessing this is what's causing the CI failures.